appbench/README.md

```diff
@@ -0,0 +1,25 @@
+# Application Performance and Benchmarking
+
+This directory contains the files for the Application Performance and Benchmarking subproject of SIG-Scalability.  The subproject's mission is:
+
+* Develop repeatable benchmarking scripts, tools, and configurations for testing the performance of applications running on Kubernetes together with stack components
```

Comment:
What do you mean by "together with stack components"?

Reply:
As in network overlays, CSI drivers, service mesh, etc.

Reply:
OK - that makes perfect sense - can you add those as examples here (to make it clear also for others)?